test/common/router/router_test.cc

```diff
@@ -2803,6 +2920,57 @@ TEST_F(RouterTest, RetryUpstreamGrpcCancelled) {
   EXPECT_TRUE(verifyHostUpstreamStats(1, 1));
 }
 
+TEST_F(RouterTest, RetryUpstreamGrpcCancelled) {
```

Comment:
Can you generally add comments to all these `TEST_F` everywhere?

Reply:
Done.

Reply:
Done.